src/components/common/BaleList.js

```diff
@@ -1,7 +1,6 @@
 import React, { Component } from 'react';
```

Comment:
This file should be under Bale and not under common. I know this is not on your part but i just realize about it